content/300-accelerate/200-getting-started.mdx

```diff
@@ -195,3 +195,54 @@ For information about which strategy best serves your application, see [Select a
 As of Prisma version `5.2.0` you can use Prisma Studio with the Accelerate connection string.
 
 </Admonition>
+
+#### Invalidating a cached query result
+
+To invalidate a cached query result, you can add tags and then use the `$accelerate.invalidate` API.
+
+:::note
+
+Programmatic cache invalidation is available with our paid plans. For more details, please see our [pricing](https://www.prisma.io/pricing#accelerate).
+
+:::
+
+To invalidate the query below:
```

Comment:
@rtbenfield , curious about your thoughts on Invalidate vs Revalidate. I know the API is `$accelerate.invalidate`, but others typically use Revalidate in this context. Does "invalidate" mean something different to us compared to the usual understanding of Revalidate?

@ShaneMaglangit curious on your thought aswell.

Reply:
What's the context behind this? I think you pinged the wrong person haha (its all good) @mhessdev

Reply:
@ShaneMaglangit, lmao, I did! Thank you for responding anyways. Dang autocomplete.

Reply:
@sneub you are the right Shane xD

Reply:
cc: @sneub is hopefully the right person.

Reply:
Invalidate -> The cache entry is practically deleted and will be repopulated only on the next request, causing a cache miss.

Revalidate -> The cache entry is proactively updated so that the next request has a cache hit.

We are doing the former (invalidate) so the naming is correct IMO.

Reply:
@mhessdev my 2cs is that invalidate is more fitting. Because revalidated to me implies, the cache is refreshed on the background. Which, AFAIK, is not the case.